packages/core/src/feedback/vue/submitFeedback.vue

```diff
@@ -81,10 +81,10 @@ export default defineComponent({
             this.error = ''
             this.isSubmitting = true
             console.log('Submitting feedback...')
-            // identifier to help us (internally) know that feedback came from either CodeWhisperer or AWS Toolkit
+            // identifier to help us (internally) know that feedback came from either Amazon Q or AWS Toolkit
+            // TODO: rework this and align with JetBrains?
             const resp = await client.submit({
-                comment:
-                    this.feedbackName === 'CodeWhisperer' ? 'CodeWhisperer onboarding: ' + this.comment : this.comment,
+                comment: this.feedbackName === 'Amazon Q' ? 'Amazon Q onboarding: ' + this.comment : this.comment,
```

Comment:
I think we removed the CW onboarding page, so this can just be "Amazon Q"
```suggestion
                comment: this.feedbackName ? `${this.feedbackName} ` + this.comment : this.comment,
```